scripts/clean_init.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+set -Eeuo pipefail
+
+echo "WARNING! This will restore your project to an intial state"
+read -p "Are you sure? y/n " -n 1 -r
+echo    # (optional) move to a new line
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    [[ "$0" = "$BASH_SOURCE" ]] && exit 1 || return 1 # handle exits from shell or function but don't exit interactive shell
+fi
+
+source ../example.env
```

Comment:
Do we want to source the example.env file? will this not overwrite our $USER.env variables?

Reply:
Had it in there in case the user did not source the env file before running, in either case as long as you are running a dev env the .env file is probably not modified. 

This is a very rough script though so can definitely change it if needed (do a find check to see if $USER.env exists for example), but wrote it with the mindset that if you run this you either way reset everything.